packages/vscode-extension/WHATISNEW.md

```diff
@@ -1,21 +1,10 @@
-## 2.6.0 - Sep 06, 2021
+## 2.7.0 - Sep 17, 2021
 
 Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
 
 ![Alt Text](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/packages/vscode-extension/img/sample.gif)
```

Comment:
the gif link is better to put under main. such like this: 
https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/sample.gif

Reply:
fixed